_health-care/_js/components/Nav.jsx

```diff
@@ -12,84 +12,84 @@ class Nav extends React.Component {
+            <h5>Military Service</h5>
             <ul className="usa-unstyled-list">
-              <li className={this.props.currentUrl === '/military-service/service-information' ? ' section-current' : ''}>
+              <li className={this.props.currentUrl === '/military-service/service-information' ? ' sub-section-current' : ''}>
                 Service
               </li>
-              <li className={this.props.currentUrl === '/military-service/additional-information' ? ' section-current' : ''}>
+              <li className={this.props.currentUrl === '/military-service/additional-information' ? ' sub-section-current' : ''}>
                 Additional Military
               </li>
             </ul>
           </div>
         </li>
-        <li role="presentation" className="step five wow fadeIn animated">
+        <li role="presentation" className={`step five wow fadeIn animated${this.props.currentUrl.startsWith('/financial-assessment') ? ' section-current' : ''}`}>
           <div>
-            <h5 className={this.props.currentUrl.startsWith('/financial-assessment') ? ' section-current' : ''}>Financial Assessment</h5>
+            <h5>Financial Assessment</h5>
             <ul className="usa-unstyled-list">
-              <li className={this.props.currentUrl === '/financial-assessment/financial-disclosure' ? ' section-current' : ''}>
+              <li className={this.props.currentUrl === '/financial-assessment/financial-disclosure' ? ' sub-section-current' : ''}>
                 Financial Disclosure
               </li>
-              <li className={this.props.currentUrl === '/financial-assessment/spouse-information' ? ' section-current' : ''}>
+              <li className={this.props.currentUrl === '/financial-assessment/spouse-information' ? ' sub-section-current' : ''}>
                 Spouse
               </li>
-              <li className={this.props.currentUrl === '/financial-assessment/child-information' ? ' section-current' : ''}>
+              <li className={this.props.currentUrl === '/financial-assessment/child-information' ? ' sub-section-current' : ''}>
                 Child
               </li>
-              <li className={this.props.currentUrl === '/financial-assessment/annual-income' ? ' section-current' : ''}>
+              <li className={this.props.currentUrl === '/financial-assessment/annual-income' ? ' sub-section-current' : ''}>
                 Annual Income
               </li>
-              <li className={this.props.currentUrl === '/financial-assessment/deductible-expenses' ? ' section-current' : ''}>
+              <li className={this.props.currentUrl === '/financial-assessment/deductible-expenses' ? ' sub-section-current' : ''}>
                 Deductible Expenses
               </li>
             </ul>
           </div>
         </li>
-        <li role="presentation" className="step six last wow fadeIn animated">
```

Comment:
Just curious why you deleted `last`? Don't we want that so there isn't a line extending down from step 6?